static/src/javascripts/projects/common/modules/commercial/dfp/dfp-obj.js

```diff
@@ -0,0 +1,4 @@
+define(function () {
```

Comment:
I'm not sure I see what your point is here, but I could move `dfp-obj` in `private` if it's clearer that way. All public modules will add themselves to the `dfp` map, but the `dfp` map itself doesn't have any dependency.